main.py

```diff
@@ -0,0 +1,15 @@
+from content_loader import QuoteLoader, ImageLoader
+from send_to_teams import TeamsNotifier
```

Comment:
Bad name for a module - if it contains notifiers, call it something like "notifiers"